### PR TITLE
add telemetry, minor fixes to GUI

### PR DESCRIPTION
**Pull request by mosabami**
## PR Checklist

- [ ] PR has a meaningful title
- [ ] Summarized changes
- [ ] This PR is ready to merge and is not **Work in Progress**
- [ ] Link to a filed issue
- [ ] Screenshot of UI changes (if PR includes UI changes)

### CODE REVIEW

**Comment by Gordonby** [comment]
Which code prettifier did you use on main.bicep? @mosabami

File: bicep/main.bicep [hunk 1]
```diff
@@ -1536,5 +1524,32 @@ resource eventGridDiags 'Microsoft.Insights/diagnosticSettings@2021-05-01-previe
...
+`---|  |----`|  |__   |  |     |  |__   |  \  /  | |  |__   `---|  |----`|  |_)  |    \   \/   /     |  .--.  ||  |__   |  |_)  | |  |     |  |  |  |  \   \/   /  |  \  /  | |  |__   |   \|  | `---|  |----`
+    |  |     |   __|  |  |     |   __|  |  |\/|  | |   __|      |  |     |      /      \_    _/      |  |  |  ||   __|  |   ___/  |  |     |  |  |  |   \_    _/   |  |\/|  | |   __|  |  . `  |     |  |
+    |  |     |  |____ |  `----.|  |____ |  |  |  | |  |____     |  |     |  |\  \----.   |  |        |  '--'  ||  |____ |  |      |  `----.|  `--'  |     |  |     |  |  |  | |  |____ |  |\   |     |  |
+    |__|     |_______||_______||_______||__|  |__| |_______|    |__|     | _| `._____|   |__|        |_______/ |_______|| _|      |_______| \______/      |__|     |__|  |__| |_______||__| \__|     |__|     */
+
+//  Telemetry Deployment
+resource telemetrydeployment 'Microsoft.Resources/deployments@2021-04-01' = if (enableTelemetry) {
+  name: telemetryId
+  location: location
+  scope: tenant()
```
[the remark this answers is not in the document]
Comment (Gordonby) [reply bicep/main.bicep:1543]: I think i've just answered my own question @mosabami 

Here is one of the errors from our CI from introducing your change. The deploying user needs significant elevation in their RBAC.
> ERROR: ***"error":***"code":"InvalidTemplateDeployment","message":"The template deployment failed with error: 'Authorization failed for template resource '3c1e2fc6-1c4b-44f9-8694-25d00ae30a3a-eastus' of type 'Microsoft.Resources/deployments'. The client 'b5a69ed4-1f7e-4855-bbcc-b3cf2dc1b8dc' with object id 'b5a69ed4-1f7e-4855-bbcc-b3cf2dc1b8dc' does not have permission to perform action 'Microsoft.Resources/deployments/write' at scope '/providers/Microsoft.Resources/deployments/3c1e2fc6-1c4b-44f9-8694-25d00ae30a3a-eastus'

**Review by Gordonby — CHANGES_REQUESTED** [review]

File: bicep/main.bicep (hunk 1 above)
[the remark this answers is not in the document]
Comment (Gordonby) [reply bicep/main.bicep:1543]: If i look at how [CARML track deployments](https://github.com/Azure/ResourceModules/blob/main/modules/Microsoft.ContainerRegistry/registries/replications/deploy.bicep#L26) then it doesn't appear we need to change the scope.

[the remark this answers is not in the document]
Comment (mosabami) [reply bicep/main.bicep]: ok @Gordonby  i changed the scope to aks

**Review by Gordonby — APPROVED** [review]
lgtm

**Pull request merged**